src/plugins/intel_cpu/src/config.h

```diff
@@ -4,9 +4,8 @@
 
 #pragma once
 
-#include <threading/ie_istreams_executor.hpp>
+#include <openvino/runtime/threading/istreams_executor.hpp>
 #include <ie_performance_hints.hpp>
```

Comment:
If we need this file, it should be moved to new API too

Reply:
yes, you are right. For properties related code, will be handled in a separated https://github.com/openvinotoolkit/openvino/pull/20022.